package.json

```diff
@@ -44,7 +44,7 @@
     "pull-paramap": "^1.2.2",
     "pull-stream": "^3.7.0",
     "ssb-bfe": "^3.7.0",
-    "ssb-box2": "^4.0.0",
+    "ssb-box2": "github:ssbc/ssb-box2#list-live",
```

Comment:
Needs merge and major release https://github.com/ssbc/ssb-box2/pull/25

Reply:
Can you merge that and update this?